deploy/kustomize/base/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+resources:
+  - ds-rezolus.yaml
+  - cr-rezolus.yaml
+  - crb-rezolus.yaml
+  - sa-rezolus.yaml
+  - sm-rezolus.yaml
+  - svc-rezolus.yaml
+
+namespace: monitoring
+
+configMapGenerator:
+  - name: rez-exporter-config
+    files:
+      - configs/exporter.toml
+  - name: rez-agent-config
+    files:
+      - configs/agent.toml
+images:
+  - name: rezolus
+    newName: rezolus
+    newTag: v5.2.2
```

Comment:
Does this get updated with each release? This feels like it could get easily missed